src/custom-types.json

```diff
@@ -236,6 +236,14 @@
   "application/x-7z-compressed": {
     "compressible": false
   },
+  "application/x-arj": {
+    "compressible": false,
+    "extensions": ["arj"],
+    "sources": [
+      "https://en.wikipedia.org/wiki/List_of_archive_formats",
+      "http://www.wussu.com/various/mimetype.htm"
```

Comment:
> `application/arj` is also referenced here and there commonly. For example, https://goo.gl/Wl1XoQ (1999).

I would guess :-1:  to this source. Has several that are incorrect... is there a revised edition book of this?

> [file(1) uses it](https://opensource.apple.com/source/file/file-37/file/magic/Magdir/archive)

Leaning towards :-1:  since that's in an archive that isn't current with apples opensource e.g. try the parent folder and try to find it as an actively referenced path in their folder view.

> Most Linux desktop relations seem to use this (for icons, launchers, etc. integration)

Confirmed with `application/x-arj` in my distro.

> lead me to `application/x-arj`

If I were to take an educated guess this is probably correct... since I can't find a registration and being of that time period. :wink:

> ... hope to hear back ...

Wish the best of luck here, seriously because I went on a hunt too with little success. :smile: ... older MIME/Content-Types are difficult to validate from that time period as it was relatively new and documentation was sparse since it was in print primarily back in the dawn of computer internet time where gophers roamed freely.